### ISSUE TITLE
Occur error when swiping on bar chart such as example 1 and 2

### ISSUE THREAD

**Issue by isoppp** [imaNNeoFighT/fl_chart#313]
**Describe the bug**
When swiping on bar chart example 1 or 2, occur error.
I guess when swiping or tapping the gaps between bars occur errors.

```
════════ Exception caught by gesture ═══════════════════════════════════════════════════════════════
The following NoSuchMethodError was thrown while handling a gesture:
The getter 'length' was called on null.
Receiver: null
Tried calling: length

When the exception was thrown, this was the stack: 
#0      Object.noSuchMethod (dart:core-patch/object_patch.dart:53:5)
#1      BarChartPainter._getNearestTouchedSpot (package:fl_chart/src/chart/bar_chart/bar_chart_painter.dart:577:43)
#2      BarChartPainter.handleTouch (package:fl_chart/src/chart/bar_chart/bar_chart_painter.dart:564:9)
#3      _BarChartState.build.<anonymous closure> (package:fl_chart/src/chart/bar_chart/bar_chart.dart:84:58)
#4      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:182:24)
...
Handler: "onCancel"
Recognizer: PanGestureRecognizer#1bc80
  debugOwner: GestureDetector
  start behavior: start
════════════════════════════════════════════════════════════════════════════════════════════════════
```

**To Reproduce**
Use bar chart example 1 and 2 and see the following video

**Screenshots**

![bug-report](https://user-images.githubusercontent.com/16318727/79807380-9083c100-831f-11ea-9c87-1266f1b89c66.gif)

**Versions**

 - which version of the Flutter are you using?
 1.18.0-dev.5.0

 - which version of the FlChart are you using?
0.9.2 ( cloned master branch today to check example in local )

**Comment by jlubeck** [comment]
I'm seeing the same problem

**Comment by imaNNeoFighT** [comment]
Fixed in [0.9.3](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.9.3),
Thanks for reporting!
